### ISSUE TITLE
conditional-display-ngFor slide non-working buttons/code reduplication

### ISSUE THREAD

**Issue by csjiang** [AngularNYC/angular-presentation#185]
[Link to slide](https://angular-presentation.firebaseapp.com/templates/conditional-display-ngfor) 
- code snippet is doubled on the left 

- comment/home buttons don't work 

- browser console logs two errors: 
1) vendor.33f9385a5ecb1ac55a41.bundle.js:148 ERROR TypeError: e.lastIndexOf is not a function
    at l (https://angular-presentation.firebaseapp.com/1.7299bec98cafbee81142.chunk.js:1:38949)
    at https://angular-presentation.firebaseapp.com/1.7299bec98cafbee81142.chunk.js:1:8157
    at Array.reduce (native)
    at t._next (https://angular-presentation.firebaseapp.com/1.7299bec98cafbee81142.chunk.js:1:8122)
    at t.__tryOrSetError (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:303411)
    at t.next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:302570)
    at t._next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:301681)
    at t.next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:301352)
    at t._emitFinal (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:131406)
    at t._emit (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:131149)
    at t._next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:130885)
    at t.next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:301352)
    at t._next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:128027)
    at t.next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:697:301352)
    at t._next (https://angular-presentation.firebaseapp.com/vendor.33f9385a5ecb1ac55a41.bundle.js:386:15923)

2) vendor.33f9385a5ecb1ac55a41.bundle.js:148 ERROR Error: Uncaught (in promise): Error: Navigation error
Error: Navigation error
    at y (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at t._next (main.ae3a73586436dcdce65f.bundle.js:1)
    at t.__tryOrUnsub (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t._next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.EEr4.t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:386)
    at vendor.33f9385a5ecb1ac55a41.bundle.js:365
    at t.invoke (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at Object.onInvoke (vendor.33f9385a5ecb1ac55a41.bundle.js:302)
    at t.invoke (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at e.run (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at polyfills.8923fee20002c5cd6ab1.bundle.js:43
    at t.invokeTask (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at Object.onInvokeTask (vendor.33f9385a5ecb1ac55a41.bundle.js:302)
    at y (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at t._next (main.ae3a73586436dcdce65f.bundle.js:1)
    at t.__tryOrUnsub (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t._next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:697)
    at t.EEr4.t.next (vendor.33f9385a5ecb1ac55a41.bundle.js:386)
    at vendor.33f9385a5ecb1ac55a41.bundle.js:365
    at t.invoke (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at Object.onInvoke (vendor.33f9385a5ecb1ac55a41.bundle.js:302)
    at t.invoke (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at e.run (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at polyfills.8923fee20002c5cd6ab1.bundle.js:43
    at t.invokeTask (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at Object.onInvokeTask (vendor.33f9385a5ecb1ac55a41.bundle.js:302)
    at y (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at l (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at polyfills.8923fee20002c5cd6ab1.bundle.js:43
    at t.invokeTask (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at Object.onInvokeTask (vendor.33f9385a5ecb1ac55a41.bundle.js:302)
    at t.invokeTask (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at e.runTask (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at a (polyfills.8923fee20002c5cd6ab1.bundle.js:43)
    at <anonymous>

**Comment by csjiang** [comment]
side buttons/keyboard input for navigating between slides also broken on this slide!

**Comment by MichaelSolati** [comment]
Duplicate of #180 I probably just phrased it wrong. Fixed in #182 just waiting on pr
